### PR TITLE
optional url for commit status

### PR DESCRIPTION
**Pull request by petey**
Requires https://github.com/screwdriver-cd/data-schema/pull/32

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -159,16 +159,22 @@ class GithubScm extends Scm {
...
+        const params = {
+            user: scmInfo.user,
+            repo: scmInfo.repo,
+            sha: config.sha,
+            state: STATE_MAP[config.buildStatus] || 'failure',
+            context: 'screwdriver'
+        };
+
+        if (config.url) {
+            params.target_url = config.url;
```
Comment (d2lam) [inline index.js]: Should add `config.url` as optional field to the comment section

**Comment by d2lam** [comment]
Small comment. the rest LGTM 👍

File: index.js [hunk 2]
```diff
@@ -187,6 +193,7 @@ class GithubScm extends Scm {
     * @param  {String}   config.path         The file in the repo to fetch
     * @param  {String}   config.token        The token used to authenticate to the SCM
     * @param  {String}   config.ref          The reference to the SCM, either branch or sha
+    * @param  {String}   [config.url]        Optional target url
```
Comment (nkatzman) [inline index.js]: are we adding this to _getFile or _updateCommitStatus?

**Comment by nkatzman** [comment]
👍

**Pull request merged**